### PR TITLE
add padel recipe

### PR DESCRIPTION
**Pull request by simonbray**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/padel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  url: http://www.yapcwsoft.com/dd/padeldescriptor/versions/PaDEL-Descriptor%20v{{ version }}.zip
+  sha256: 9cfba80da884d67e79dc9777fbe842f84ee08b5457251eda37e5a2d8cfd80637
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+    - openjdk
```
Comment (marcelotrevisani) [inline recipes/padel/meta.yaml]: Do you really need ``Java``?

File: recipes/padel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python >=3
+    - pip
+  run:
+    - openjdk
+    - python >=3
+    - pip
```
Comment (marcelotrevisani) [inline recipes/padel/meta.yaml]: Do you really need ``pip`` to run this project?

File: recipes/padel/build.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+mkdir -p $PREFIX/bin
+
+cp $RECIPE_DIR/padel.py $PREFIX/bin/padel
+cp -r * $PREFIX
```
Comment (marcelotrevisani) [inline recipes/padel/build.sh]: doesn't this project have a setup.py?

Comment (marcelotrevisani) [reply recipes/padel/build.sh]: You are just copying the Python file, there are a lot of ``.jar`` on this package. Shall you install all of them as well?

Comment (simonbray) [reply recipes/padel/build.sh]: Hi @marcelotrevisani, the package is written in Java. I also wrote a small python wrapper script to invoke the jar file.

File: recipes/padel/meta.yaml (hunk 2 above)
Comment (simonbray) [reply recipes/padel/meta.yaml]: You're right, pip is not required.

File: recipes/padel/build.sh (hunk 3 above)
Comment (marcelotrevisani) [reply recipes/padel/build.sh]: Ah ok, I see

**Pull request merged**